prog/postgres/postgres_server_nexus.rb

```diff
@@ -149,13 +149,28 @@ def before_run
     refresh_walg_credentials
 
     when_initial_provisioning_set? do
-      hop_configure
+      hop_configure_prometheus
     end
 
     vm.sshable.cmd("sudo -u postgres pg_ctlcluster 16 main reload")
     hop_wait
   end
 
+  label def configure_prometheus
+    vm.sshable.cmd("sudo userdel -rf prometheus || true")
+    vm.sshable.cmd("sudo adduser --disabled-password --gecos '' prometheus")
+    vm.sshable.cmd("sudo userdel -rf ubi_monitoring || true")
+    vm.sshable.cmd("sudo adduser --disabled-password --gecos '' ubi_monitoring")
+
+    web_config = <<CONFIG
+tls_server_config:
+  cert_file: /dat/16/data/server.crt
+  key_file: /dat/16/data/server.key
+CONFIG
+    vm.sshable.cmd("sudo -u prometheus tee /home/prometheus/web-config.yml > /dev/null", stdin: web_config)
```

Comment:
how will you limit who can connect to the web server? will `web-config.yml` contain additional settings in the future?

Reply:
Yes, that is correct. Subsequent commits will add additional configs for:
1. Setting up authentication
2. Lmiting the number of metrics we emit.